.github/workflows/git-secrets.yml

```diff
@@ -1,14 +1,18 @@
-name: gitleaks
+name: Git Secrets
+
 on:
-  workflow_dispatch:
   push:
-  pull_request:
+  workflow_dispatch:
```

Comment:
Removing pull_request from this one. I might be convinced but currently the branch rules (just for this repo afaik) say a PR isn't mergable until gitleaks-scan step is finished satisfactorily. 

I wonder if we can make it a precommit thing and then just a pull_request one enforced by the branch rules. What do you think?